esx_service/vmdk_ops.py

```diff
@@ -777,8 +779,9 @@ def datastore_path_exist(datastore_name):
 
 def get_datastore_name(datastore_url):
     """ Get datastore_name with given datastore_url """
+    logging.debug("get_datastore_name: datastore_url=%s", datastore_url)
     datastore_name = vmdk_utils.get_datastore_name(datastore_url)
-    if datastore_name is None or not datastore_path_exist(datastore_name):
+    if not datastore_path_exist(datastore_name):
```

Comment:
is it guaranteed that datastore_name is non-None. Because datastore_path_exist() expects that.